### ISSUE TITLE
Alternative Download Source, Please

### ISSUE THREAD

**Issue by rosenfeldamir** [activitynet/ActivityNet#91]
tl;dr : 
It will be a *huge* service if someone with the dataset can put it on an alternative download site - Academic Torrents, some aws s3 storage, etc. Perhaps something like they did here: https://github.com/cvdfoundation/ 

details:
The current download options (google drive, baidu) simply don't work for me. I've been trying for several weeks. The google-drive files either fail due to exceeded quota or the link expires before I managed to download and I can't login to baidu, not for lack of trying (based outside of china, tried the phone sms option, doesn't work...).
I'm currently stuck before even getting started, only because I can't download the dataset. I am absolutely sure I'm not the only one.

**Comment by alpargun** [comment]
I agree that it is now not possible to download the data set through this repo. In my case, the downloaded video files cannot be opened through a video player such as VLC. So, I am currently using the following repo to download the Kinetics data set:
https://github.com/cvdfoundation/kinetics-dataset

However, some functions in this repo, ActivityNet, can still be useful for moving the video files to corresponding folders according to their labels in order to obtain the data set structure Facebook Research models use since the repo I linked only downloads the files to train, validation and test directories.
